gradio/utils.py

```diff
@@ -1098,7 +1098,7 @@ def is_in_or_equal(path_1: str | Path, path_2: str | Path) -> bool:
 
 
 @document()
-def set_static_paths(paths: list[str | Path]) -> None:
+def set_static_paths(paths: str | Path | list[str | Path]) -> list[str | Path]:
```

Comment:
Its safer to accept a str explicitly because otherwise if a user passes in a str accidentally, every individual character will be treated as a separate "file"